docs/guide/seo.rst

```diff
@@ -0,0 +1,57 @@
+================================
+Search Engine Optimization (SEO)
+================================
+
+What is search engine optimization?
+-----------------------------------
+
+Search engines drive a significant portion of site traffic for docs that publish as public-facing websites.
+Search engine optimization (SEO) describes the means and methods used to have your web pages display for a set of desired search terms.
+
+The work of SEO demands familiarity with the ever-changing rules of the algorithms search engines use to select and arrange web sites to return for any particular query.
+These algorithms change frequently, so the requirements for achieving the best optimization also change frequently.
+
+Fortunately, good tech writing already does most of the standard practices required for excellent SEO.
+
+- Write good headings
+- Make your docs skimmable by breaking content into sections under appropriate headings
+- Focus on good information architecture that provides quality navigation and site hierarchy
+- Include page metadata
+- Work on making sure your documentation meets standards for accessibility
+
+Resources
+---------
+
+Search engine documentation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Review the search engine documentation guides:
+
+- Google's `Search Central <https://developers.google.com/search/docs>`__
+- Google's `How Search Works <https://www.google.com/search/howsearchworks/how-search-works/ranking-results/>`_ guide
+- Bing's `Webmaster Guidelines <https://www.bing.com/webmasters/help/webmasters-guidelines-30fba23a>`__
+
+WTD resources on SEO
```

Comment:
duplicates the topics page, do we want to link to that page instead of duplicating those links?

Reply:
I tried in an earlier commit, but couldn't figure out how to link to the specific heading.
I'd rather link there if someone else knows how to get that done.

Reply:
I think `/topics/#search` should work

Reply:
I don't see Sphinx interlinking used anywhere else, but gonna give it a shot, as that's what we use in the MinIO docs.

Reply:
> I think `/topics/#search` should work

You would like to think. Something else must also be broken, but this link is giving me headaches.